docs/release.md

```diff
@@ -28,11 +28,11 @@ We use the [npm-version](https://docs.npmjs.com/cli/v6/commands/npm-version) com
 To create a new release, follow these steps:
 
 1. Make sure your `master` branch is up-to-date, and you have the most recent git tags in your repo: `git pull upstream master --tags`.
-1. Make sure your working tree is clean.
+1. Make sure your working tree passes all checks (indicated by the green marker) and is clean.
```

Comment:
`1. Make sure that the latest commit on `master` is passing CI on GitHub Actions, and that you are sitting on the same commit locally (i.e., `git show` or `git log` and compare the git SHA).`